openfecwebapp/templates/legal_advisory_opinion_landing.html

```diff
@@ -0,0 +1,50 @@
+{% extends "layouts/main.html" %}
+{% import 'macros/legal-search.html' as legal_search %}
+{% import 'macros/breadcrumbs.html' as breadcrumb %}
+
+{% set breadcrumb_links=[(cms_url + '/legal-resources', 'Legal resources')] %}
+
+{% block title %}
+  {% if query %}
+  Searching {{ document_type_display_name }} for &ldquo;{{ query }}&rdquo;
+  {% else %}
+  Search {{ document_type_display_name }}
+  {% endif %}
+{% endblock %}
+
+{% block body %}
+<header class="page-header slab slab--primary">
+  {{ breadcrumb.breadcrumbs(document_type_display_name, breadcrumb_links) }}
+</header>
+
+<section class="main__content--full data-container__wrapper">
+  {% include 'partials/legal-filter.html' %}
+  <div class="content__section data-container__body dataTables_wrapper">
+  <div class="results-info results-info--simple">
+    <h1 class="results-info__title" id="advisory-opinions">Advisory opinions
+    </h1>
+    </div>
+      <p class="u-padding--top">Advisory opinions are official Commission responses to questions about how federal campaign finance law applies to specific, factual situations.
+      </p>
+      <p class="u-padding--top">This archive contains advisory opinions from 1975 to the present. It also contains documents related to advisory opinions — such as requests, drafts and public comments — from 1990 to the present.
+      </p>
```

Comment:
Can you clean up the tags here, the open tag starts inline, but the close tag is on it's own line. I'd prefer it be consistent, both on their own line or both inline.